### PR TITLE
Add Day of Dragons

### PR DESCRIPTION
**Pull request by Greelan**
Adds Day of Dragons dedicated server.

Notes:
- Supplies Game.ini with all possible settings in UI
- Regex to be added

### CODE REVIEW

**Comment by IceOfWraith** [comment]
Pls stop. I can't keep up!

**Comment by Greelan** [comment]
Lol, take your time. I still have a couple of yours to look at. This one is pretty much there but more finessing needed

**Comment by Shinynecrid** [comment]
> Pls stop. I can't keep up!

Lmao

**Comment by Greelan** [comment]
Based on feedback and logs from a user on Discord, there seems to be nothing logged by the server for user join/leave/chat. So nothing for the regex. 

The user otherwise confirmed that the server is operating as expected. So this template is ready for review.

File: day-of-dragonsmetaconfig.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[	{
+        "ConfigFile":"Dragons/Saved/Config/WindowsServer/Game.ini", 
+        "AutoMap":true,
+        "ConfigType":"ini",
+		"ConfigFormatRegex":"^(?<key>.+?)=(?<value>.*?)$"
```
Comment (IceOfWraith) [inline day-of-dragonsmetaconfig.json:5]: Minor spacing issue on both Format lines.

**Review by IceOfWraith — CHANGES_REQUESTED** [review]
Looks good. Don't have the game to test, but as mentioned it was tested by another user.

File: day-of-dragonsmetaconfig.json (hunk 1 above)
Comment (Greelan) [reply day-of-dragonsmetaconfig.json]: Thanks, fixed

**Review by IceOfWraith — APPROVED** [review]
Very nice! Are you up to 50 yet? :p

**Pull request merged**